contracts/truefi/TrueLender.sol

```diff
@@ -58,11 +58,11 @@ contract TrueLender is ITrueLender, Ownable {
     uint256 public minApy;
     uint256 public maxApy;
 
-    // How many votes in prediction market
-    uint256 public participationFactor;
```

Comment:
we cannot change the order of storage variables by deleting this line.

The pattern we use for deprecating variables is to add _Depricated to the end of the variable name. Please add back any variables deleted here, and add new variables to the bottom of the storage section.

Reply:
e.g. participationFactor_Depricated, riskAversion_Depricated

renaming storage variables is okay, changing the order or removing storage variables is not

Reply:
Renaming is actually what I did. I figured that we need 2 variables of size uint256 and want to get rid of 2 of the same type at once.